classes/book.rb

```diff
@@ -0,0 +1,12 @@
+require '../item'
+class Book < Item
+  def initialize(publisher, cover_state, publish_date = Time.year)
+    super(publish_date)
```

Comment:
- [ ] Extend `archived` from super method
 ```suggestion
   super(publish_date, archived: archived, id: id)
```

Reply:
can you explain why archived and id should be  extended its not part of the uml diagram